llama-index-integrations/readers/llama-index-readers-pebblo/tests/test_readers_pebblo.py

```diff
@@ -0,0 +1,69 @@
+from llama_index.core.readers.base import BaseReader
+from llama_index.readers.pebblo import PebbloSafeReader
+from pathlib import Path
+from typing import Dict
+from pytest_mock import MockerFixture
+from llama_index.readers.file import CSVReader
+
+
+class MockResponse:
+    def __init__(self, json_data: Dict, status_code: int):
+        self.json_data = json_data
+        self.status_code = status_code
+
+    def json(self) -> Dict:
+        return self.json_data
+
+
+def test_class():
+    names_of_base_classes = [b.__name__ for b in PebbloSafeReader.__mro__]
+    assert BaseReader.__name__ in names_of_base_classes
+
+
+def test_empty_filebased_loader(mocker: MockerFixture) -> None:
+    """Test basic file based csv loader."""
+    mocker.patch.multiple(
+        "requests",
+        get=MockResponse(json_data={"data": ""}, status_code=200),
+        post=MockResponse(json_data={"data": ""}, status_code=200),
+    )
+
+    file_path = f"{Path().resolve()}/test_empty.csv"
```

Comment:
instead of committing these test files in the repo, can we just write them temporarily and then clean them up?

Reply:
Added test setup and teardown to handle file creation and deletion for test.